source/common/upstream/upstream_impl.cc

```diff
@@ -730,6 +730,13 @@ ClusterInfoImpl::ClusterInfoImpl(
     idle_timeout_ = std::chrono::hours(1);
   }
 
+  if (config.common_http_protocol_options().has_max_connection_duration()) {
```

Comment:
Please revert all changes in this file. We can add this back later if/when implemented.

Reply:
I was following suggestion about partial implementation in https://github.com/envoyproxy/envoy/pull/8591#discussion_r337322325.
I'm ok with either of approaches :)

Reply:
I think by partial implementation @htuch meant downstream vs. upstream, not half of upstream. :) Thank you!

Reply:
I think I just misread 'delivers partial value'. I'll revert these changes.

Reply:
reverted.